packages/editor/src/components/document-bar/style.scss

```diff
@@ -66,7 +66,7 @@
 }
 
 .editor-document-bar__shortcut {
-	color: $gray-800;
+	color: $gray-700;
```

Comment:
This may need to remain $gray-800, or be #6d6d6d to meet 4.5:1 contrast. https://contrast.tools is a great way to test.

Other than that, ship it!